### PR TITLE
fix jekins failure

### PR DESCRIPTION
**Pull request by dding3**
Fix https://github.com/intel-analytics/arda-docker/issues/563

Seems keras softmax doesn't support tensor with dim>3 while tf.keras does.

### CODE REVIEW

**Comment by dding3** [comment]
jekins pass. http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-PR-Dllib-Validation/188/
